.gitignore

```diff
@@ -17,6 +17,7 @@ __pycache__/
 .idea/
 .venv/
 
+jwtRS512.key
```

Comment:
please add `**/*.key`. This will prevent git from tracking any private key accidentally.